### ISSUE TITLE
Turning off Don't alter window class name created by sandboxed programs automatically enables other settings

### ISSUE THREAD

**Issue by silvestron** [sandboxie-plus/Sandboxie#783]
## Describe the bug

Disabling the setting called *Don't alter window class name created by sandboxed programs* from the settings menu in Sandboxie Plus automatically turns on *Force usage of custom dummy Manifest files (legacy behaviour)* and *Emulate sandboxed window station for all processes* after closing the settings window.

Enabling *Don't alter window class name created by sandboxed programs* again automatically turns off the other options.

## To Reproduce

1. Go to Advanced Options -> Miscellaneous -> Compatibility
2. Check/uncheck *Don't alter window class name created by sandboxed programs*
3. Confirm settings

## Expected behavior

I don't know what's the expected behavior but this doesn't happen if you manually edit the ini file. Even if this was an expected behavior the user doesn't see that disabling a setting automatically turned on other settings until they open the settings again or open the ini file.

## Screenshots

![sbie](https://user-images.githubusercontent.com/68440738/114561487-e575c100-9c6d-11eb-8a9b-0ecea9366119.png)

## System details and installed software

- Windows 10 Pro 20H2
- Sandboxie Plus 0.7.4

## Sandboxie configuration

Settings affected in the ini file

When *Don't alter window class name created by sandboxed programs* is ON:

```
OpenWinClass=#
```

When *Don't alter window class name created by sandboxed programs* is OFF:

```
PreferExternalManifest=y
UseSbieWndStation=y
BlockPassword=n
```

**Comment by SiNONiMiTY** [comment]
Can confirm, this only happens after updating to 0.7.4

**Comment by SiNONiMiTY** [comment]
Can confirm, this only happens after updating to 0.7.4

EDIT
It seems to automatically enable these settings whenever a change is done in the "Advanced" section.
> PreferExternalManifest=y
> UseSbieWndStation=y
> BlockPassword=n

**Comment by SiNONiMiTY** [comment]
Can confirm, this only happens after updating to 0.7.4

It seems to automatically enable these settings whenever a change is done in the "Advanced" section.
> PreferExternalManifest=y
> UseSbieWndStation=y
> BlockPassword=n

**Comment by SiNONiMiTY** [comment]
@DavidXanatos Can confirm, this only happens after updating to 0.7.4

It seems to automatically enable these settings whenever a change is done in the "Advanced Options" section.
> PreferExternalManifest=y
> UseSbieWndStation=y
> BlockPassword=n

**Comment by DavidXanatos** [comment]
that is fixed in 0.7.7
